### PR TITLE
Improve README and add linter check

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Using the TLDR linter the page will be linted for errors, the page given will then be staged onto a new branch, committed and pushed to origin
```
Comment (CleanMachine1) [inline README.md]:
```suggestion
Using the TLDR linter, the page will be linted for errors, the page given will then be staged onto a new branch, committed and pushed to origin
```

**Review by 258204 — APPROVED** [review]
Thanks @CleanMachine1! :D

**Comment by CleanMachine1** [comment]
Did you make sure it works?

**Comment by 258204** [comment]
> Did you make sure it works?

I did a basic test, but didn't check cases of the linter failing or not being installed. It doesn't exit properly in those cases, I'll commit a fix in a few minutes.

**Pull request merged**